scripts/call_back.py

```diff
@@ -0,0 +1,19 @@
+
+def even_numver_list(limit,cb):
```

Comment:
Its always a good practice to follow PEP8 and Pylint.
Configure it and add PEP8 and Pylint score.

If you are using Pycharm configure in the following way:

https://stackoverflow.com/questions/38134086/how-to-run-pylint-with-pycharm